packages/preview/uninsubria-thesis/0.1.0/README.md

```diff
@@ -0,0 +1,201 @@
+# Tesi uninsubria
+
+DISCLAIMER: the template was forked from https://github.com/roland-KA/clean-dhbw-typst-template. there may be some inaccuracies in the docs and source.
+
+A (modernized) [Typst](https://typst.app/) template for documents like Bachelor theses, project documentation etc. 
+
+You can see an example of how the template looks in this [PDF file](https://github.com/sbOogway/tesi-uninsubria/blob/main/template/main.pdf).
```

Comment:
This link is broken. I would recommend making a copy of this file at the root of the package, called `example.pdf` for example, and link to `./example.pdf` directly.